login-frontend/package.json

```diff
@@ -27,10 +29,10 @@
     "eslint-plugin-react-hooks": "^4.6.0",
     "events": "^3.1.0",
     "os-browserify": "^0.3.0",
-    "parcel": "^2.9.2",
```

Comment:
How dare you

Reply:
I am very sorry! I reverted the changes. Instead of using a different bundler, I fetch the endpoint from the backend. [87bf7a5](https://github.com/KILTprotocol/opendid/pull/36/commits/87bf7a52d248c2a8eee2ca00c49dcdb0788dba01)

Reply:
😃